src/mono/browser/runtime/types/index.ts

```diff
@@ -595,6 +598,46 @@ export type APIType = {
     localHeapViewF64: () => Float64Array;
 }
 
+export type DiagnosticsAPIType = {
```

Comment:
nit: Naming based on other type names

```suggestion
export type DiagnosticAPIType = {
```